Example/Controllers/PointsSettingsTableViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  PointsSettingsTableViewController.swift
+//  MiniApp_Example
+//
+//  Created by Laubengaier, Timotheus | MTSD on 2021/07/16.
+//  Copyright © 2021 Rakuten Group, Inc. All rights reserved.
+//
```

Comment:
Please remove the header comment

Reply:
oh yeah, I just removed the header comment and added a changelog feature entry